src/management-system-v2/components/auth.tsx

```diff
@@ -31,22 +31,30 @@ export const getCurrentEnvironment = cache(
   ) => {
     const { userId } = await getCurrentUser();
 
-    if (!activeEnvironment) {
+    // Use hardcoded environment /my/processes for personal spaces.
+    if (activeEnvironment === 'my') {
+      // Note: will be undefined for not logged in users
+      activeEnvironment = userId;
+    }
+
+    // FIXME: This might fail for personal spaces now, we should always define
+    // the active space id.
+    /*if (!activeEnvironment) {
```

Comment:
As you mentioned, this breaks some server actions, could you update them?

Reply:
They now should all include a spaceId.